cbor/src/test/java/com/fasterxml/jackson/dataformat/cbor/ParserWithJsonOrgSampleTest.java

```diff
@@ -47,6 +47,16 @@ public void testJsonSampleDoc() throws IOException
         verifyJsonSpecSampleDoc(cborParser(data), false, true);
     }
 
+    public void testJsonSampleDocStringref() throws IOException
+    {
+        byte[] data = cborDoc(new CBORFactory().enable(CBORGenerator.Feature.STRINGREF),
```

Comment:
Could you change this to use Builder? This merges cleanly with 3.0 where factories are immutable.

Reply:
I've switched this and the one below to use a builder.